static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -98,6 +97,18 @@ export class SentryAppExternalForm extends Component<Props, State> {
       this.debouncedOptionLoad(field, input, resolve);
     });
 
+  getSubmitEndpoint() {
+    const {sentryAppInstallationUuid} = this.props;
+    switch (this.props.element) {
+      case 'alert-rule-action':
+        // TODO(leander): Send request to the correct endpoint
```

Comment:
@NisanthanNanthakumar -- Just want to double check, there isn't an endpoint yet to save the alert rule right?

Reply:
It needs to populate the parent field state, my bad -- i'll look into this and reopen the PR when it's good to go